deps/spidershim/include/v8.h

```diff
@@ -478,13 +480,13 @@ struct BarrierMethods {
 template <class T>
 class PersistentBase {
  public:
-  V8_INLINE void Reset();
+  void Reset();
```

Comment:
The compiler was throwing errors on inlining functions without bodies. This might be a mac/linux compiler difference.